go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/articulate/terraform-provider-ohdear
 
-go 1.22
+go 1.23
```

Comment:
I think the go version field changed slightly in 1.22, so we might need to upgrade minor versions as well. We had to do that in another project at least

Reply:
Updated to 1.23.0 - lints and tests still running locally.